webpack/assets/javascripts/react_app/components/MemoryAllocationInput/__tests__/MemoryAllocationInput.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { mount } from '@theforeman/test';
+import { testComponentSnapshotsWithFixtures } from '../../../common/testHelpers';
+import MemoryAllocationInput from '../MemoryAllocationInput';
+
+const props = {
+  label: 'Memory',
+};
+
+const warningProps = {
+  value: 11264,
+};
+
+const errorProps = {
+  value: 21504,
+};
+
+const fixtures = {
+  'should render with default props': props,
+  'should render with warning message': warningProps,
+  'should render with error message': errorProps,
+};
+
+describe('MemoryAllocationInput', () => {
+  describe('rendering', () => {
+    testComponentSnapshotsWithFixtures(MemoryAllocationInput, fixtures);
+  });
+
+  test('GB change', () => {
+    const component = mount(<MemoryAllocationInput />);
+    expect(component.render()).toMatchSnapshot();
```

Comment:
creating a snapshot by using `mount` creates a huge snapshot that we don't actually need.
Using `testComponentSnapshotsWithFixtures` for creating snapshots is enough (and it uses `shallow` rendering)